### ISSUE TITLE
Support receiving file share intents

### ISSUE THREAD

**Issue by robinlinden** [evilcorpltd/aTox#699]
Right now only text is supported as that was a lot easier and added a lot of the plumbing needed for this. The difference with files (right now) is that we need to only allow it to online contacts.

**Comment by theAkito** [comment]
I just tested sharing a `"text/plain"` in Android SDK 29 and this app did not show up in the Sharesheet.

**Comment by robinlinden** [comment]
If you're using the latest release from GPlay/FDroid I think it's missing accepting shares altogether. :( It was implemented just after the last release. Will be part of the next one, though! Or you can give the nightly a try if you're curious: https://robinli.eu/f/atox-0.7.0-pre10.apk

**Comment by theAkito** [comment]
Okay, I see now. I downloaded the latest version from F-Droid today, to test sharing from an app I am working on. Was surprised to not see it work. But if it's going to be released at some point, then I can use this Tox client for testing after that.

**Comment by robinlinden** [comment]
Yeah, I'm hoping to make a release soon. :x Working on getting audio calls working well on all Android devices right now, but after that it's release time!
